### PR TITLE
#160951982 Fix error when checkbox clicked

### PR DESCRIPTION
**Pull request by matthewacha**
#### What does this PR do?
This PR fixes error returned when the checkbox is clicked

#### Description of Task to be completed?
Currently, an error is returned when you click the checkbox on new request modal, however, this error should not be returned

#### How should this be manually tested?
* Clone this repo branch  - `git clone -b bg-checkbox-state-cached-160371607 https://github.com/andela/travel_tool_front.git`
* Switch to the repo directory - cd travel_tool_front
* Install dependencies and start the server - make start
* Set up the server by following this guide
* Navigate to localhost:5000 in a browser
* sign in using your andela email address
* Click the `New request` button
* Add your personal details, check the checkbox and submit form
* Navigate back to the `New request` button
* your personal details should be in the form inputs

#### What are the relevant pivotal tracker stories?
[160951982](https://www.pivotaltracker.com/story/show/160951982)

#### unchecked
image

##### checked
image

#### Questions:

### CODE REVIEW

File: src/views/Requests/index.jsx [hunk 1]
```diff
@@ -49,15 +49,6 @@ export class Requests extends Base {
     openModal(true, 'edit request');
   }
 
-  componentDidUpdate(){
```
Comment (albert85) [inline src/views/Requests/index.jsx]: @matthewacha why did you remove this line of code

Comment (matthewacha) [reply src/views/Requests/index.jsx]: I had a sync with Collins concerning why he had added that line and agreed to remove it because it only took effect after refreshing the modal. refer to this [bug story](https://www.pivotaltracker.com/story/show/160813365)

Comment (albert85) [reply src/views/Requests/index.jsx]: @matthewacha thanks for the reply. can you update the label of your PR to  be `PEER_REVIEW`
